interfaces/openfga/v0/README.md

```diff
@@ -0,0 +1,67 @@
+# `openfga`
+
+## Usage
+
+This relation interface describes the expected behaviour of any charm claiming to be able to interact with a OpenFGA.
+
+In most cases, this will be accomplished using the [openfga library](https://github.com/canonical/cs-openfga/blob/main/charms/openfga-k8s/lib/charms/openfga_k8s/v0/openfga.py), although charm developers are free to provide alternative libraries as long as they fulfil the behavioural and schematic requirements described in this document.
```

Comment:
Is there a reason for the linked repo to be private?

Reply:
no, it should be public as agreed a few weeks ago

Reply:
it was made public last week